uaclient/entitlements/realtime.py

```diff
@@ -0,0 +1,53 @@
+from typing import Tuple
+
+from uaclient import messages, util
+from uaclient.entitlements import repo
+from uaclient.entitlements.base import IncompatibleService
+from uaclient.types import MessagingOperationsDict
+
+REALTIME_KERNEL_DOCS_URL = "https://ubuntu.com/realtime"  # TODO
+
+
+class RealtimeKernelEntitlement(repo.RepoEntitlement):
+    name = "realtime-kernel"
+    title = "Real-Time Kernel"
+    description = "Realtime Kernel"
```

Comment:
@nathan-j-hart @EdoardoBarbieriCanonical What should the 3-8 word description of the realtime kernel be?

This is what is displayed in the output of `ua status`. It could be something like:
```
realtimer-kernel       yes       enabled         Ubuntu Kernel with PREEMPT_RT patches
```
Or maybe something like
```
realtimer-kernel       yes       enabled         Kernel designed for consistent low-latency
```
Or more or less technical as you see fit.

Reply:
I'd go with the first option